### PR TITLE
Move deprecated examples

### PR DESCRIPTION
**Pull request by paul-szczepanek-arm**
Mbed cli doesn't use .mbedignore and imports contained libs (by design).

### CODE REVIEW

**Comment by pan-** [comment]
@paul-szczepanek-arm we need the `.mbed` file at the root of the project or it will confuse mbed cli.

**Comment by paul-szczepanek-arm** [comment]
> @paul-szczepanek-arm we need the `.mbed` file at the root of the project or it will confuse mbed cli.

thanks, fixed

File: BLE_Thermometer/mbed-os [hunk 1]
```diff
@@ -0,0 +1 @@
+C:/ws/github/pan/mbed-os
```
Comment (pan-) [inline BLE_Thermometer/mbed-os]: That must be wrong.

Comment (paul-szczepanek-arm) [reply BLE_Thermometer/mbed-os]: I have no clue how this happened

File: BLE_Thermometer/deprecated/BLE_Thermometer/.mbed [hunk 2]
```diff
@@ -1 +0,0 @@
-ROOT=.
```
Comment (pan-) [inline BLE_Thermometer/deprecated/BLE_Thermometer/.mbed]: Could you move that file and not delete it ?

**Pull request merged**